db/seedData/generatePGdata.js

```diff
@@ -0,0 +1,106 @@
+const faker = require('faker');
+
+const placeTypes = ['bakery', 'bar', 'cafe', 'restaurant', 'meal_delivery', 'meal_takeaway'];
+
+function generateRestaurantDataBatch(round) {
+  if (round % 100 === 0) {console.log(round);}
+  if (round > 1000) {
+    return Promise.resolve(null);
+  }
+  let data = [];
+  for (var x = 0; x < 10000; x++) {
+    let restId = (10000 * round) + x;
+    let restaurantRow = generateRestaurantObj(restId);
+    data.push(restaurantRow);
+  }
+  return Promise.resolve(data);
```

Comment:
Curious about your use of ``Promise.resolve()`` throughout the data generation script since it is seemingly synchronous code. Is that done to make the generation functions easily compatible with pg-promise task and transactions? I'm not very familiar with the tasks and transactions so wasn't sure if that is just a pattern they use that I'm not familiar with.

Reply:
I use Promise.resolve() because it is the pattern that was in the 'Data Imports' example in the pg-promise wiki. In this case, I believe its used so that you can call .then to use the pgp.helpers to insert the data that was generated.